backend/src/main/scala/cromwell/backend/wdl/ReadLikeFunctions.scala

```diff
@@ -1,27 +1,16 @@
-package cromwell.backend
+package cromwell.backend.wdl
 
-import java.nio.file.Path
-
-import better.files._
+import cromwell.backend.MemorySize
 import wdl4s.expression.WdlStandardLibraryFunctions
 import wdl4s.parser.MemoryUnit
-import wdl4s.types._
+import wdl4s.types.{WdlArrayType, WdlFileType, WdlObjectType, WdlStringType}
 import wdl4s.values._
-import wdl4s.TsvSerializable
 
 import scala.language.postfixOps
 import scala.util.{Failure, Success, Try}
-import scala.language.implicitConversions
-
-trait WdlStandardLibraryImpl extends WdlStandardLibraryFunctions {
-
-  def toPath(str: String): Path
 
-  override final def fileContentsToString(path: String): String = toPath(path).toAbsolutePath.contentAsString
-
-  private def writeContent(baseName: String, content: String): Try[WdlFile] = {
-    Try(WdlFile(writeTempFile(tempFilePath, s"$baseName.", ".tmp", content)))
-  }
+trait ReadLikeFunctions extends FileSystems { this: WdlStandardLibraryFunctions =>
```

Comment:
What's the difference between the `PureFunctions` trait above and this trait? As in we have two similar implementations of the `sub` function defined in both traits